swarm/network/stream/streamer_test.go

```diff
@@ -1114,3 +1121,170 @@ func TestRequestPeerSubscriptions(t *testing.T) {
 		}
 	}
 }
+
+/*
+TestGetSubscriptionsRPC sets up a simulation network of 16 nodes,
+starts the simulation, waits for SyncUpdateDelay in order to kick off
+stream registration, then tests that there are subscriptions.
+*/
+func TestGetSubscriptionsRPC(t *testing.T) {
```

Comment:
I'd rather we remove this test, provided that subscriptions are tested elsewhere. The RPC API just iterates over all subscriptions and returns them, I don't think adding a simulation test that takes 2sec. on my machine brings much value.

I think this is actually introducing technical debt by adding a simulation test here.